lib/OpenQA/Utils.pm

```diff
@@ -399,5 +400,19 @@ sub check_download_whitelist {
     return ();
 }
 
+sub notify_workers {
+    my $ipc = OpenQA::IPC->ipc;
```

Comment:
oh, right. yeah, when I was gonna make it a function I was going to invent `IPC::Common` or something like that for it to live in...